package.json

```diff
@@ -45,9 +45,12 @@
     "react-bootstrap": "^0.30.10",
     "react-chartjs-2": "^2.0.5",
     "react-dom": "^15.4.2",
+    "react-dom-factories": "^1.0.1",
     "react-rangeslider": "^2.0.1",
     "react-redux": "^5.0.2",
     "react-router": "^3.0.2",
+    "react-selectize": "^3.0.1",
+    "react-transition-group": "^1.2.0",
```

Comment:
Why is react-transition-group added? How does it benefit us?

Reply:
https://github.com/furqanZafar/react-selectize#install

Reply:
In general, I'm against adding three dependencies to our project for something this simple (HTML input and selection field). Every dependency increases our bundle size.

Reply:
Writing the component by hand seems like a poor use of time to save a 71 kilobytes (1,322,461 bytes for my bundle vs 1,250,851 bytes for groupUI) especially given that these dependencies also provide single-select fuzzy search controls (for, e.g. groups editing UI).